tools/testoperator/src/commands/bench/mod.rs

```diff
@@ -115,6 +115,8 @@ pub(crate) async fn run(args: &DatasetTestArgs) -> anyhow::Result<RowCounts> {
 
     crate::metrics::TEST_DURATION
         .record((metrics.finished_at - metrics.started_at).try_into()?, &[]);
+    crate::metrics::MAX_MEMORY_USAGE.record(max_memory * 1024.0, &[]);
```

Comment:
```suggestion
    crate::metrics::PEAK_MEMORY_USAGE.record(peak_memory * 1024.0, &[]);
```

Can we call this peak?